README.md

```diff
@@ -1,329 +1,14 @@
 # ASTRA-sim 2.0
-ASTRA-sim is a distributed machine learning system simulator, developed as a joint collaboration between Georgia Tech, Meta, and Intel.
+[ASTRA-sim](https://astra-sim.github.io/) is a cycle-accurate distributed training simulator developed by Intel, Meta, and Georgia Tech. It enables the systematic study of challenges in modern deep learning systems, allowing for the exploration of bottlenecks and the development of efficient training methodologies for large DNN models across diverse future platforms.
```

Comment:
I prefer to refer to ASTRA-sim as a distributed machine learning system simulator rather than a cycle-accurate training simulator. ASTRA-sim is not limited to training purposes alone; it can be used for a variety of multi-GPU simulations. Although ASTRA-sim utilizes cycles as its simulation unit, 'cycle-accurate' is not the most appropriate term to describe it.

Reply:
agreed!